ddtrace/ext/span_kind.go

```diff
@@ -0,0 +1,33 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+package ext
+
+// span_kind values are set per span following the opentelemetry standard
+// falls under the values of client, server, producer, consumer, and internal
```

Comment:
> falls under the values of client, server, producer, consumer, and internal

I'm not sure I understand what this means. Can we possibly clarify slightly and link to the otel standard?

Also, Nit: let's get rid of the newline between this comment and const.

Reply:
This is the [link](https://github.com/open-telemetry/opentelemetry-specification/blob/9fa7c656b26647b27e485a6af7e38dc716eba98a/specification/trace/api.md#spankind) to the otel documents explaining what span kind is and the possible values. This .go file should explain the different values possible for span.kind, but if there are more changes necessary, I can add those